tests/Google/Ads/GoogleAds/Lib/V1/GoogleAdsClientBuilderTest.php

```diff
@@ -139,6 +141,30 @@ public function testBuildFailsWithoutOAuth2Credential()
             ->build();
     }
 
+    public function provideInvalidProxyURIs()
```

Comment:
Nit: Move this to below testBuildFailsWithInvalidProxyUri.

Reply:
Done.